app/views/nsm/disbursements/edit.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title t('.page_title') %>
 
 <% content_for(:back_link) do %>
-  <%= link_to t('helpers.back_link'), "", id: "back-button", class: 'govuk-back-link', data: { turbo: 'false' } %>
+  <%= link_to t('helpers.back_link'), nsm_claim_disbursements_path(claim, anchor: item.id), class: 'govuk-back-link' %>
```

Comment:
This does behave weirdly if you were on adjusted_nsm_claim_disbursements_path and clicked through to this page from there, but unless we start storing state somehow to detect which table page you were most recently on, I think it's unavoidable, and this is still an improvement.

Reply:
Well as I said in the PR body, we can do something similar to what `:back` does and use the referrer header to determine which URL to go back to.

I'll see how complex that ends up being as it may satiate the best of all worlds then

Reply:
Have gone for a much simpler method to just return the backlink path for a claim from an item which will also let us switch on potential other views in the future.

Think this is the best of all worlds now, unless there are any _other_ places that specifically link to a work item or disbursement.